js/i18n.js

```diff
@@ -73,6 +73,26 @@ function searchForResources(search_url) {
       var time = getSingleItem(params, 'time');
       var manifest = getSingleItem(params, 'manifest');
       var languages = getArrayItem(params, 'lc');
+      
+      var searchFieldArr = [];
+      if(full_text) searchFieldArr.push(full_text);
+      if(repo_name) searchFieldArr.push('repo:'+repo_name);
+      if(user_name) searchFieldArr.push('user:'+user_name);
```

Comment:
True, but they won't get put in the search field unless they type user=asdf or whatever in the URL as well. We should have the search bar displaying what exactly was searched, so if they should be removed here, we should also disable being able to put user=asdf and other params in the URL, for they need to see what is beings searched on in the search area.